Description: This patch adds an XPM icon to the KVIrc package.
  Debian menu files support icons but just in XPM format. This patch creates
  the XPM icon file.
  .
  This patch is Debian specific.
Author: Robin Verduijn <robin@debian.org>
Last-Update: 2009-04-03 16:14:48 +0000
Origin: vendor, http://hg.debian.org/hg/pkg-kde/kde-extras/kvirc/raw-file/tip/debian/patches/05_xpmicon.patch
Forwarded: no
---
 data/icons/32x32/CMakeLists.txt |    3 
 data/icons/32x32/kvirc32x32.xpm |  295 ++++++++++++++++++++++++++++++++++++++++
 2 files changed, 298 insertions(+)

Index: b/data/icons/32x32/CMakeLists.txt
===================================================================
--- a/data/icons/32x32/CMakeLists.txt
+++ b/data/icons/32x32/CMakeLists.txt
@@ -19,6 +19,9 @@ IF(UNIX)
 		# Assume linux
 		INSTALL(FILES ${iconapps} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/32x32/apps/)
 		INSTALL(FILES ${iconmime} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/icons/hicolor/32x32/mimetypes/)
+		# Install xpm for ${CMAKE_INSTALL_PREFIX}/share/menu/kvirc
+		FILE(GLOB files "*.xpm")
+		INSTALL(FILES ${files} DESTINATION ${CMAKE_INSTALL_PREFIX}/share/pixmaps/)
 	ENDIF()
 ELSEIF(WIN32)
 	INSTALL(FILES ${iconapps} DESTINATION ${CMAKE_INSTALL_PREFIX}/icons/32x32/apps/)
Index: b/data/icons/32x32/kvirc32x32.xpm
===================================================================
--- /dev/null
+++ b/data/icons/32x32/kvirc32x32.xpm
@@ -0,0 +1,295 @@
+/* XPM */
+static char *kvirc3[] = {
+/* columns rows colors chars-per-pixel */
+"32 32 257 2",
+"   c #000000",
+".  c #09050A",
+"X  c #0C0A0C",
+"o  c #070408",
+"O  c #0F0810",
+"+  c #130A14",
+"@  c #171218",
+"#  c #1F1420",
+"$  c #201E21",
+"%  c #2A1A2B",
+"&  c #251526",
+"*  c #331C35",
+"=  c #361E38",
+"-  c #242024",
+";  c #2D282D",
+":  c #2A242A",
+">  c #2F2930",
+",  c #372538",
+"<  c #3E2E40",
+"1  c #6D3B08",
+"2  c #4A294C",
+"3  c #47274A",
+"4  c #4E2B50",
+"5  c #4F275E",
+"6  c #483C49",
+"7  c #532E55",
+"8  c #542C5B",
+"9  c #592F57",
+"0  c #56315A",
+"q  c #59315B",
+"w  c #4F2661",
+"e  c #552C62",
+"r  c #572B67",
+"t  c #5D3361",
+"y  c #5E3268",
+"u  c #613664",
+"i  c #653669",
+"p  c #653868",
+"a  c #6A3A6D",
+"s  c #643F66",
+"d  c #623372",
+"f  c #6D3C71",
+"g  c #6A367A",
+"h  c #713E74",
+"j  c #753D7C",
+"k  c #79440A",
+"l  c #69586A",
+"z  c #744077",
+"x  c #754078",
+"c  c #79427C",
+"v  c #794F7B",
+"b  c #6F626F",
+"n  c #7A607B",
+"m  c #6E3C83",
+"M  c #763681",
+"N  c #7C4582",
+"B  c #79418A",
+"V  c #7D4F84",
+"C  c #7E7080",
+"Z  c #8D3E00",
+"A  c #993F00",
+"S  c #8D4000",
+"D  c #894808",
+"F  c #944400",
+"G  c #984100",
+"H  c #885615",
+"J  c #9E5422",
+"K  c #905031",
+"L  c #92611C",
+"P  c #9C6B21",
+"I  c #967533",
+"U  c #A54800",
+"Y  c #AB4B00",
+"T  c #A65409",
+"R  c #B24F00",
+"E  c #B35400",
+"W  c #B95700",
+"Q  c #BD5A01",
+"!  c #B9610B",
+"~  c #B9621D",
+"^  c #A96A21",
+"/  c #A77825",
+"(  c #BA7B37",
+")  c #835343",
+"_  c #915B48",
+"`  c #975A5A",
+"'  c #8A4972",
+"]  c #885E66",
+"[  c #94665C",
+"{  c #8F6F67",
+"}  c #836266",
+"|  c #97656A",
+" . c #AA704E",
+".. c #B37351",
+"X. c #A96358",
+"o. c #A07D6A",
+"O. c #C25A00",
+"+. c #CD5800",
+"@. c #C45700",
+"#. c #CF6100",
+"$. c #D36200",
+"%. c #DF6900",
+"&. c #D5680C",
+"*. c #E56400",
+"=. c #E06A00",
+"-. c #E86700",
+";. c #EA6900",
+":. c #EE7000",
+">. c #F27100",
+",. c #FD7600",
+"<. c #FE7A02",
+"1. c #FF7D0A",
+"2. c #823A8B",
+"3. c #804684",
+"4. c #85448B",
+"5. c #854A8A",
+"6. c #8A4D8E",
+"7. c #8D4D93",
+"8. c #88568F",
+"9. c #924C9B",
+"0. c #945699",
+"q. c #98568C",
+"w. c #975CA1",
+"e. c #9E56A1",
+"r. c #9850AE",
+"t. c #9E58A3",
+"y. c #9551A4",
+"u. c #9650B0",
+"i. c #9B54B3",
+"p. c #866787",
+"a. c #8F6894",
+"s. c #976993",
+"d. c #9861A1",
+"f. c #A057A5",
+"g. c #A159A6",
+"h. c #A55BAA",
+"j. c #AA5EAF",
+"k. c #A25ABB",
+"l. c #AB5EB0",
+"z. c #A658BA",
+"x. c #A967AD",
+"c. c #AE61B3",
+"v. c #AF6BB5",
+"b. c #A872AA",
+"n. c #AD7AB3",
+"m. c #B063B5",
+"M. c #B564BB",
+"N. c #B869BE",
+"B. c #B469B9",
+"V. c #B574B9",
+"C. c #BD6BC2",
+"Z. c #B769C7",
+"A. c #BE74C3",
+"S. c #BD79C1",
+"D. c #BA75C9",
+"F. c #C16FC7",
+"G. c #C06ED4",
+"H. c #C171C6",
+"J. c #C272C9",
+"K. c #C57ACA",
+"L. c #C27AC6",
+"P. c #9F833F",
+"I. c #AE8224",
+"U. c #BA8532",
+"Y. c #BB9657",
+"T. c #B2837C",
+"R. c #BEA262",
+"E. c #D98B32",
+"W. c #DBAD33",
+"Q. c #E1961E",
+"!. c #FF830B",
+"~. c #FF8519",
+"^. c #FF881C",
+"/. c #FF8816",
+"(. c #FF8C24",
+"). c #FF8E28",
+"_. c #FF9120",
+"`. c #FF912F",
+"'. c #FF963A",
+"]. c #FF993D",
+"[. c #FE9833",
+"{. c #F6A908",
+"}. c #F7A927",
+"|. c #FFBA2A",
+" X c #C78441",
+".X c #CA8D53",
+"XX c #CA827B",
+"oX c #CBA157",
+"OX c #D5BD58",
+"+X c #CBB656",
+"@X c #CABF6B",
+"#X c #D4AA6F",
+"$X c #C9A967",
+"%X c #EA9D43",
+"&X c #FF9C44",
+"*X c #E1BF4F",
+"=X c #E7BF5A",
+"-X c #FFA241",
+";X c #FEA556",
+":X c #FFA759",
+">X c #E0B662",
+",X c #EEB374",
+"<X c #FEB36D",
+"1X c #FFB565",
+"2X c #FFB672",
+"3X c #D6C46B",
+"4X c #EDC74B",
+"5X c #FFCF41",
+"6X c #FFDB41",
+"7X c #FFD15A",
+"8X c #E8C957",
+"9X c #E6C261",
+"0X c #FFD862",
+"qX c #FFDD72",
+"wX c #FFDC79",
+"eX c #F3C66B",
+"rX c #FFE046",
+"tX c #FFE476",
+"yX c #FFF36A",
+"uX c #FFFB79",
+"iX c #B19587",
+"pX c #B988BC",
+"aX c #B3A189",
+"sX c #BD8EC0",
+"dX c #CE8781",
+"fX c #D4B892",
+"gX c #FEBD83",
+"hX c #CC92D0",
+"jX c #D094D5",
+"kX c #FFC387",
+"lX c #FDCC96",
+"zX c #FEDD83",
+"xX c #FFD29C",
+"cX c #E5D89C",
+"vX c #E9D0A8",
+"bX c #FFCCA0",
+"nX c #FDD3AB",
+"mX c #FFE787",
+"MX c #FFE490",
+"NX c #FFED97",
+"BX c #FFEA9A",
+"VX c #FFF087",
+"CX c #FFF28E",
+"ZX c #FFEFA6",
+"AX c #F6E5C8",
+"SX c #FFE2C5",
+"DX c #FEE4CA",
+"FX c #FCEBCC",
+"GX c #F5E2C4",
+"HX c #FCEBD2",
+"JX c #F7EDD4",
+"KX c #FFF0CE",
+"LX c #FAF1DB",
+"PX c #FFF3E1",
+"IX c #FFFAF2",
+"UX c #FFFFF9",
+"YX c None",
+/* pixels */
+"YXYXYXYXYXYXYXYXYXYXYXYX  X - $ $ $ O   YXYXYXYXYXYXYXYXYXYXYXYX",
+"YXYXYXYXYXYXYXYXYXYXYXYX  C jXhXhXjXb   YXYXYXYXYXYXYXYXYXYXYXYX",
+"YXYXYXYXYXYX    YXYXYX    p.V.V.V.v.n     YXYXYX    YXYXYXYXYXYX",
+"YXYXYXYX    ; X       X b n.V.hXL.V.b.l o       + :     YXYXYXYX",
+"YXYXYX    > sXpX< > a.pXS.L.L.K.K.H.M.m.V.p.; 6 b.b.-     YXYXYX",
+"YXYXYX  X a.d.0.d.d.d.v.K.K.K.K.J.J.F.M.9.M 4.4.M M V @     YXYX",
+"YXYX  : s.vXJXGXGXAXAXiXJ.J.J.J.F.C.C.l.iXJXLXLXHXLXfX8.@   YXYX",
+"YXYX  X n.T.LXIXIXIXIXnX9.Z.k.k.z.z.z.M KXUXUXPXHXKX} 0.o   YXYX",
+"YXYXYX  , s.lXDXDXSXSXxX| dX%X}.}.].XX| SXSXnXnXlX,Xj >     YXYX",
+"YXYXYX  % D...xXbXbXgX<X X7XwXwX0X5X{..XkXgXgX<X1X_ y.@   YXYXYX",
+"YXYXYX  v J.q.-X2X<X;X&X].R.MXzXzX7XI.;X;X;X&X].[.y 5.s     YXYX",
+"YXYX  o v.J.G.~ -X&X'.).(.( ZXMXmXtX^ ].'.(.(._.J B 3.6.    YXYX",
+"      0 A.C.Z.' /.`.).^.^.&.cXBXMX3X&.).^.~.~.!.9 N c N 3       ",
+". q a t.N.C.M.k.Q /.(.!.1.<.Y.NXVXP.<.1.<.1.<.T m c x x h q 7 o ",
+"@ C.j.M.M.N.l.z.K <.!.<.,.,.! NXCXT >.<.,.,.,.) m x h h a a c + ",
+"+ l.j.c.c.m.h.i.Q.Q <.>.>.>.*.3X3X+.:.:.:.>.E >Xg h a a a p x O ",
+"+ j.g.j.h.h.g.u.}./ ;.=.=.=.*.U.I =.=.%.=.=./ eXd a a i i t h O ",
+"+ j.g.h.h.f.f.r.E.4XY $.$.$.$.T k $.#.#.#.U 8X#Xd i i u t q a O ",
+"o , 7 N h.f.f.y.X.6XL @.O.O.O.O.E O.O.O.@.L yX| y u t t q , = o ",
+"  o   * f.e.0.7.B |.8XG E E E E E E E E A OXtXr t q q t %       ",
+"YXYX    N 7.6.4.B ` 6XL E E E E E E E E H uX] e q q q q     YXYX",
+"YXYXYX  3 6.3.3.c j  .W.U E E E E E W Y +Xo.5 0 q q q *     YXYX",
+"YXYXYX  X N N c x j g [ k F Z Z Z S F 1 } 5 8 q q q q .   YXYXYX",
+"YXYXYX  @ c c x z z f g d [ oX=X9X$X{ e 0 0 q q q q q +   YXYXYX",
+"YXYX  o u c x h h a f a i y r e w 8 8 q q q q q q q t 4     YXYX",
+"YXYX  X u x h h a a p i t t t q 9 q q q 8 q q q q q q 4 .   YXYX",
+"YXYXYX  . q h f p u p u q q q q q q q q q q q q q t 2 o     YXYX",
+"YXYXYX    . q 7 + . * 7 t q q q 0 q q q 7 * . + 4 4 o     YXYXYX",
+"YXYXYXYX    o           & 7 q q q q 2 #           o     YXYXYXYX",
+"YXYXYXYXYXYX    YXYXYX    = q q q t *       YXYX      YXYXYXYXYX",
+"YXYXYXYXYXYXYXYXYXYXYXYX  % t t t u &   YXYXYXYXYXYXYXYXYXYXYXYX",
+"YXYXYXYXYXYXYXYXYXYXYXYX    o o o o     YXYXYXYXYXYXYXYXYXYXYXYX"
+};
